src/Compilers/CSharp/Portable/Symbols/Wrapped/WrappedLocalSymbol.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    /// <summary>
+    /// Represents a local that is based on another local.
+    /// When inheriting from this class, one shouldn't assume that 
+    /// the default behavior it has is appropriate for every case.
+    /// That behavior should be carefully reviewed and derived type
+    /// should override behavior as appropriate.
+    /// </summary>
+    internal abstract class WrappedLocalSymbol : LocalSymbol
+    {
+        public abstract LocalSymbol UnderlyingLocal { get; }
+        public override Symbol ContainingSymbol => UnderlyingLocal.ContainingSymbol;
+        public override TypeWithAnnotations TypeWithAnnotations => UnderlyingLocal.TypeWithAnnotations;
+        public override RefKind RefKind => UnderlyingLocal.RefKind;
+        public override ImmutableArray<Location> Locations => UnderlyingLocal.Locations;
+        public override ImmutableArray<SyntaxReference> DeclaringSyntaxReferences => UnderlyingLocal.DeclaringSyntaxReferences;
+        public override string Name => UnderlyingLocal.Name;
+        public override bool IsImplicitlyDeclared => UnderlyingLocal.IsImplicitlyDeclared;
+        protected override Symbol OriginalSymbolDefinition => (object)UnderlyingLocal.OriginalDefinition == UnderlyingLocal ? this : UnderlyingLocal.OriginalDefinition;
```

Comment:
>(object)UnderlyingLocal.OriginalDefinition == UnderlyingLocal ? this : UnderlyingLocal.OriginalDefinition; [](start = 62, length = 106)

I think this should always be ```this```.

Reply:
That's the default behavior, will remove this override.

---
In reply to: [329713644](https://github.com/dotnet/roslyn/pull/38769#discussion_r329713644) [](ancestors = 329713644)